recipe/meta.yaml

```
@@ -77,8 +84,9 @@ outputs:
       script: python -m pip install .[hub] --no-deps --ignore-installed -vv
     requirements:
       host:
+        - __unix
+        - python >=3.7
         - pip
-        - python
       run:
         - {{ pin_subpackage(name + "-base", exact=True) }}
```

Not sure about this one
```suggestion
        - __unix
        - {{ pin_subpackage(name + "-base", exact=True) }}
```